.github/ISSUE_TEMPLATE/new_product_suggestion.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Product Suggestion
+about: Suggest a new product to add into endoflife.date
+title: ''
+labels: 'new-product'
+assignees: ''
+
+---
+
+`Full and Short Name of Suggested Product`
+
+Let us know its full long name and if exist its short name ( ie PowerShell , pwsh )
+
+
+`Does this product have LTS versions ? Date intervals between each LTS versions ?`
+
+ie. A current release is a release that occurs between LTS releases. Current releases can contain critical fixes,
+innovations, and new features. A current release is supported for three months after the next release (current or LTS).
+
+`What is the Website of Suggested Product and its Version information ?`
+
+ie. https://github.com/PowerShell/PowerShell/tags
+
+`Additional context`
+
+Add any other context or screenshots about the product which you want us to add endoflife.date .
```

Comment:
```suggestion
Add any other context or screenshots about the product which you want us to add endoflife.date.
```

Reply:
This one is not suitable because it will messup link think something like abc.com. is not abc.com thus that dot must be seperated or we need to add another text after url so we can put . at the end without space

Reply:
If you have abc.com at the end of a sentence, you end with a full stop, with no space.

And it's not being rendered as a link, full stop or not:

![image](https://user-images.githubusercontent.com/1324225/140112218-f1e36f73-a68e-47fe-b038-27c95ca1a29e.png)

Reply:
i fully removed dot